src/irmin-pack/inode.ml

```diff
@@ -81,7 +81,7 @@ struct
 
     type inode = { index : int; hash : H.t }
     type inodes = { seed : int; length : int; entries : inode list }
-    type v = Values of (step * value) list | Inodes of inodes
+    type v = Values of (step * value) list | Branch of inodes
```

Comment:
if you want to use `Branch` (which I found a bit confusing because "Git branches") you should also use `Leaf` instead of `Values` for consistency.

Reply:
`Fork` instead?

Reply:
I'd say either `Tree of 'a | Value of 'b` or `Branch of 'a | Leaf of 'b` (and I tend to prefer the first one)